web/controllers/game_controller.ex

```diff
@@ -0,0 +1,11 @@
+defmodule Yggdrasil.GameController do
+  use Yggdrasil.Web, :controller
+
+  alias Yggdrasil.Game
+
+  def index(conn, _params) do
+    games = Repo.all Game
```

Comment:
So for example

```Elixir
# admin removing a character
if Yggdrasil.Player.online(character_id) do
  :ok = Yggdrasil.Player.shutdown(character_id)
end
Yggdrasil.Character.remove(character_id)

# User trying to do it
case Yggdrasil.Player.online(character_id) do
  true -> # notify user they can't
  false -> Yggdrasil.Character.remove(character_id)
end
```
Yggdrasil.Character.remove(character_id) can do checks to ensure that it's not in active player or running and error if so. It can be that simple right, then the web uses other apis to shutdown the player or what ever.

However if we did this

```Elixir
# admin removing a character
if Yggdrasil.Player.online(character_id) do
  :ok = Yggdrasil.Player.shutdown(character_id)
end
char = Yggdrasil.Repo.get Character, character_id
Yggdrasil.Repo.delete character

# User trying to do it
case Yggdrasil.Player.online(character_id) do
  true -> # notify user they can't
  false -> 
    char = Yggdrasil.Repo.get Character, character_id
    Yggdrasil.Repo.delete character
end
```
So two code paths may have direct access to remove character, and we have to rely that they will both check the active players and more testing rather duplication of tests in my opinion.

Don't know I just was thinking that if we did what the first form we could then test out Yggdrasil.Character.remove() to ensure it only succeeds if the constraints are met and for example it's not actively being used. This way we can ensure one code path and consitency.

That's the reason I'm opposed to direct Repo access.